package.json

```diff
@@ -1,11 +1,16 @@
 {
   "name": "@genialis/resolwe",
-  "version": "0.2.4",
+  "author": "Genialis Inc.",
+  "version": "1.0.0",
   "description": "Resolwe frontend libraries",
+  "license" : "SEE LICENSE IN LICENSE",
```

Comment:
What's up with this, shouldn't this be some kind of an identifier for the license?

Reply:
Apache License (2.0)

Reply:
This is valid entry according to https://docs.npmjs.com/files/package.json#license

Since there's `LICENSE` file there, why not just reference it?

Reply:
Updated.

Reply:
Sure but it probably makes it easier to parse automatically? Otherwise you need to actually extract the `LICENSE` and perform "license detection" on it :-)